src/compiler/diagnosticMessages.json

```diff
@@ -3118,5 +3118,9 @@
     "Implement inherited abstract class": {
         "category": "Message",
         "code": 90007
+    },
+    "Adding a tsconfig.json file will help organizing projects with both TypeScript and JavaScript files. Learn more at https://aka.ms/tsconfig": {

```

Comment:
Consider changing to 

"will help organize projects that contain both TypeScript and JavaScript files."